0400-office-data_management.Rmd

```diff
@@ -22,22 +22,23 @@ Exporting the PSCIS data to a gpkg is a great way to view everything spatially (
 
 To see the site photos, look in Mergin and/or Onedrive directories. 
 
-### PSCIS phase 1 data QA
+### PSCIS Phase 1 Data QA
 
 Here is a rough list of things to check at each site for phase 1 QA:
 
   + Correct spelling.
-  + Clear concise comments.
+  + Clear, concise comments.
   + Add any relevant info from climate change assessments comments or pictures to the main `assessment_comments` column.
   + Backwatered, is the box ticked, does it align with the photos?
   + Embeddedment, is the box ticked, does it align with the photos?
-  + Correct number of `moti_chris_culvert_id`. Check photos and comments to see how many culverts there are. If it's a major Moti structure than include the `chris_hwy_structure_road_id` instead. Beware that MoTi datasets are really bad and often incorrect.
   + Beaver activity, is the box ticked, does it align with the photos?
   + Fish observations during assessment, is the box ticked?
+  + Correct number of `moti_chris_culvert_id`. Check photos and comments to see how many culverts there are. If it's a major Moti structure than include the `chris_hwy_structure_road_id` instead. Beware that MoTi datasets are really bad and often incorrect.
   + Is the crossing type correct, check photos.
+  + Is the assessment type (Phase 1, Phase 2, or Reassessment) correct? For clarification see [PSCIS Phase 1]. 
   + Are the 6 photos correct? Barrel, inlet, outlet, upstream, downstream, and road. If a photo is truly missing (check photo folder first) then add a `no_photo` photo to the site folder. 
   + Is there an important landowner nearby? If so, could we collaborate?
-  + Has there been past restoration work at the site? Search stream name in [EcoCat](https://a100.gov.bc.ca/pub/acat/public/welcome.do). If yes and there is a link to a report, add relevant info and url link to the assessment comments. Additionally, add the reference to `Zotero/region/` and then add the citation key to the `my_citation_key` column.
+  + Has there been past restoration work at the site? Search the stream name in [EcoCat](https://a100.gov.bc.ca/pub/acat/public/welcome.do). If yes and there is a link to a report, add relevant info and url link to the assessment comments. Additionally, add the reference to `Zotero/region/` and then add the citation key to the `my_citation_key` column.
```

Comment:
also search google and relevant local databases like the [Skeena Salmon Data Portal](https://data.skeenasalmon.info/)

Reply:
No just past restoration work but any documents that are relevant for the reporting of this stream. 
 Even search key reports like Skeena Salmon and Their Habitats....   Add to zotero with correct/complete bib conventions (if not there yet)and citation key to `my_citation_key1`, `my_citation_key12, `my_citation_key3` columns (or whatever their correct names are)